sessions/nowcasting.qmd

```diff
@@ -391,4 +391,19 @@ What do you think of the nowcast now? How would you know you had the wrong delay
 - In practice, you would likely compare the nowcast to the observed data and if they did not match you would consider whether the delay distribution was the cause.
 :::
 
+# Going further
+
+## Methods in practice
+
+- Wolffram et al., [Collaborative nowcasting of COVID-19 hospitalization incidences in Germany](https://doi.org/10.1371/journal.pcbi.1011394) compares the performance of a range of methods that were used in a nowcasting hub and investigates what might explain performance differences.
```

Comment:
Do you want to link to some of the actual methods? 

In terms of what is in this session it is similar to the epinow2 nowcasting model, there is also baselinenowcast. 

All the other methods that come to mind are joint approaches and so should be in the following joint session imo